pkg/controllers/resources/persistentvolumes/translate.go

```diff
@@ -113,6 +113,11 @@ func (s *persistentVolumeSyncer) translateUpdate(ctx context.Context, vPv *corev
 		updated.Spec.Capacity = vPv.Spec.Capacity
 	}
 
+	if !equality.Semantic.DeepEqual(pPv.Spec.ClaimRef, vPv.Spec.ClaimRef) {
+		updated = translator.NewIfNil(updated, pPv)
+		updated.Spec.ClaimRef = vPv.Spec.ClaimRef
```

Comment:
Isn't that the wrong reference here, the claimref is not translated in this context correctly, or am I mistaken?